src/Router/Silex.php

```diff
@@ -115,4 +116,16 @@ public function getResponse($content = '', $statusCode = 200, $headers = array()
     {
         return new HttpFoundationResponse($content, $statusCode, $headers);
     }
+
```

Comment:
you can move your BinaryFileResponse logic by branching in the getResponse `if $file exist`